libqtile/widget/wallpaper.py

```diff
@@ -40,6 +41,7 @@ class Wallpaper(base._TextBox):
         ("label", None, "Use a fixed label instead of image name."),
         ("option", "fill", "How to fit the wallpaper when wallpaper_command is"
             "None. None, 'fill' or 'stretch'."),
+        ("update_interval", 5, "Update interval in seconds, if none, it won't update"),
```

Comment:
Doesn't this change the widget to update every five seconds by default? That feels quite jarring. Can we make this `None` to match the old behavior?

Reply:
That is because when you use this widget, you want to know if the auto change is working or not.

Reply:
Can you write a test for that? I'd prefer to keep the widget's behavior as it is now.

Reply:
I'm sorry I don't know how to write a test. I didn't find any topic on writing tests on the website.
And a little complain here if you don't like you can ignore.
I saw the same widget was committed 2 years ago but didn't get merged. This is not how an open source project should be running.  I guess it's the same reason why openweather widget is on github, but not merged also, despite no one will use yahoo weather over open weather map. Anyway, as a irrelevant member of a small community, I don't have the right to criticize anything.

Reply:
> I didn't find any topic on writing tests on the website.

Trying putting `python tests` into your favourite search engine, and consider reading some of the tests in our 'tests' folder. We use [pytest](https://docs.pytest.org/en/latest/) for testing.

>  saw the same widget was committed 2 years ago but didn't get merged.

Why wasn't it merged? Was it waiting for something? Feel free to post a comment on it if you want to draw attention to it.

> I guess it's the same reason why openweather widget is on github, but not merged

Looks to me like we're waiting for the submitter to respond. Sometimes people go for weeks/months without working on qtile, because it's a hobby project and people have lives. What would you have us do? Anyway that seems completely unrelated to your PR and writing a test.

> Anyway, as a irrelevant member of a small community, I don't have the right to criticize anything.

I don't think you are irrelevant, and indeed you do have the right to criticise anything we do, but your comment sounded more like frustration taken out on us for some reason that I cannot get from your rant. If you do have general problems with the way that Qtile is managed then please feel free to post an issue expressing your concerns and what we can do to improve.